packages/ui/src/location.ts

```diff
@@ -170,16 +170,16 @@ export function setLocationStorageKey (storageKey: string): void {
   locationStorageKeyId = storageKey
 }
 
-export function navigate (location: PlatformLocation, store = true): boolean {
+export function navigate (location: PlatformLocation, store: boolean | { replace: true } = true): boolean {
```

Comment:
May be better have ?
```suggestion
export function navigate (location: PlatformLocation, store: 'store'|'replace' = 'store'): boolean {
```

Reply:
Just wanted a quick backward compatible change to avoid digging all existing use cases. But after your comment had to do it. And I don't see any existing calls with explicit last argument. So we can make it even simpler:
```typescript
export function navigate (location: PlatformLocation, replace = false): boolean {
```
And store unconditionally

Reply:
nice :-)